examples/GUI/GUIGlobe.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GUIGlobe
+ * Description: Classe pour créer un menu.
+ */
+
+
+function GUIGlobe() {
+    //Constructor
+
+    this.gui = new dat.GUI( { autoPlace : false});
```

Comment:
I will probably need some help on that point... I guess there is some work to do on webpack.config et package.json but I'm not very comfortable with the package.json...